.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "go.testTags": "all",
```

Comment:
accidental vscode directory commit?

Reply:
Bah yeah, but actually helpful to have there ... think this is caused by the upgrade tool committing everything